### PR TITLE
[ADD] New module for cart preview

### PR DESCRIPTION
**Pull request by LeartS**
New module that backports a feature from v9 which shows
a cart preview in a popover.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
If this a backport, you should respect git commit history. See https://github.com/odoomrp/odoomrp-wip/wiki/Migration-to-OCA as an example for making the migration.

**Comment by LeartS** [comment]
It's an "adjusted" backport of this commit: https://github.com/odoo/odoo/commit/a2b7678f10a9dd7c3c3a15ff1e444a6514e72252 I've made some modifications. How should I proceed?

**Comment by pedrobaeza** [comment]
OK, I see that it's an extraction. You should mention the commit and the author of the commit on the README.

File: website_sale_cart_preview/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Credits
+=======
+
+This module is a backport of the v9 implementation of this feature.
+
+Contributors
+------------
+
+* Leonardo Donelli <donelli@webmonks.it>: Initial author of the module
```
Comment (pedrobaeza) [inline website_sale_cart_preview/README.md]: Here it should be original extraction and adaptation, but we don't usually put this info: commits tell us this.

**Comment by LeartS** [comment]
@pedrobaeza is it ok now?

File: website_sale_cart_preview/controllers/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from . import main
```
Comment (pedrobaeza) [inline website_sale_cart_preview/controllers/__init__.py:2]: Put encoding

File: website_sale_cart_preview/controllers/main.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+from openerp import http
```
Comment (pedrobaeza) [inline website_sale_cart_preview/controllers/main.py:3]: Put encoding

**Comment by pedrobaeza** [comment]
It lacks translations.

The rest :+1: (functional also)

**Comment by LeartS** [comment]
I'll move this PR to [OCA/e-commerce](https://github.com/OCA/e-commerce)

**Comment by pedrobaeza** [comment]
Thanks

**Pull request closed without merging**